tests/acceptance/literature/test_literature_new_form.py

```diff
@@ -116,7 +121,39 @@ def test_literature_create_thesis_manually(login):
 
     create_literature.go_to()
     assert create_literature.submit_thesis(input_data).has_error()
-    _check_back_office(input_data)
+    _check_back_office()
+
+
+def test_record_files_creation(login):
+    create_literature.go_to()
+    assert create_literature.import_and_submit_arxiv_by_id('hep-th/9711200').has_error()
+
+    expected_data = {
+        'title': (
+            'The Large N Limit of Superconformal Field Theories and '
+            'Supergravity'
+        ),
+        'user': 'admin@inspirehep.net',
+        'authors-0-name': 'Maldacena, Juan',
+        'abstract': (
+            'We show that the large $N$ limit of certain conformal field '
+            'theories'
+        )
+    }
+
+    _check_back_office(expected_data)
+    holding_panel_literature_detail.go_to()
+    submission_number = top_navigation_page.current_url().split('/')[-1]
+
+    from invenio_records.api import RecordMetadata
+
+    record = [record for record in RecordMetadata.query.all()
+              if 'acquisition_source' in record.json and record.json['acquisition_source']['submission_number'] == submission_number][0]
```

Comment:
So, in order to get a record with files you create a new acceptance test (which requires you to modify some existing code in the acceptance test suite) in which you submit a record through the UI, you click all the buttons you need to click, then you look which URL you get and finally you get this record by using the id you parsed from the URL.

Perhaps there is a simpler way?

Reply:
We decided to do this in order not to generate the whole workflow process in order to submit a new record. We can discuss another way if there is something easier.